### PR TITLE
Updated data cleaning script, model selection script, update env file, add tuning script, add draft model results

### PR DESCRIPTION
**Pull request by nicovandenhooff**
A summary of what this PR includes:

- Minor edits to data preprocessing script
  - Changed revenue column to 0s and 1s (needed for xgboost and sklearn)
  - Filled nans in feature engineering caused by zero divison
  - Formatted script to be pep8 compliant

- Cleaned up model selection script and added draft results to results folder
- Added XGBoost to env
- Wrote and added model tuning script
- Added draft model results from tuning script to results folder

### CODE REVIEW

**Comment by ytz** [comment]
Small issue: Not sure why, but the confusion matrix png seems to crop out the left label (can't see 'True Label')

**Comment by ytz** [comment]
Attempt to reopen

**Review by ytz — COMMENTED** [review]
Small issue: Not sure why, but the confusion matrix png seems to crop out the left label (can't see 'True Label')

Also TIL 'format on save' in vscode
